pom.xml

```diff
@@ -2361,7 +2372,7 @@
             <groupId>${hive.group}</groupId>
             <artifactId>hive-service-rpc</artifactId>
           </exclusion>
-          <!-- parquet-hadoop-bundle:1.8.1 conflict with 1.12.3 -->
+          <!-- parquet-hadoop-bundle:1.8.1 conflict with 1.12.4 -->
```

Comment:
Rather than "upgrading" the version in the comment (that does not really say much) I'd reword it to the following (and get rid of the version):

```
parquet-hadoop-bundle:1.8.1 conflict with [the name of the artifact it conflicts with]
```